### PR TITLE
Bump up fireperf version and update Changelog.

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**
1. Bump up Fireperf version from 20.0.0 to 20.0.1 for M97 release.
2. Remove `currentReleasedVersion` to avoid confusion.
3. Fix flaky tests in AppStateMonitorTest.java.

Note: this is not a blocker for M97 release.

### CODE REVIEW

**Comment by jeremyjiang-dev** [comment]
device-check-changed  have `writeLeafNodeRemoveLeafNodeWaitForEvents` and `writeMultipleLeafNodesRemoveOneLeafNodeWaitForEvents` are failing under `firebase-database`, which are not modified by this PR.

**Review by jfcong — APPROVED** [review]

**Comment by visumickey** [comment]
Thanks for cleaning up all the pending release updates as well. Change LGTM!

**Review by visumickey — APPROVED** [review]

File: firebase-perf/CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,49 @@ Refer [GMaven](https://maven.google.com/web/index.html?q=firebase-perf#com.googl
 
 ## Released
 
+## v20.0.1 (M97)
+
+#### Android library
+
+*   {{feature}} Added Firebase console URL logging to logcat.
```
Comment (visumickey) [inline firebase-perf/CHANGELOG.md:30]: Firebase performance logs now contain URL to the see the performance metrics on the Firebase Console.

Comment (jeremyjiang-dev) [reply firebase-perf/CHANGELOG.md]: Done.

**Comment by jeremyjiang-dev** [comment]
@rlazo Would you like to take a look at the error from `firebase-common`?
```
com.google.firebase.platforminfo.FirebasePlatformLoggingTest > test_watch_atNotHighEnoughApiLevel FAILED
    com.google.common.truth.AssertionErrorWithFacts at FirebasePlatformLoggingTest.java:82
```
It seems like there is an extra space after "android-platform/"

**Comment by rlazo** [comment]
Hey Jeremy #2700 should fix this. Thanks!

**Comment by jeremyjiang-dev** [comment]
> Hey Jeremy #2700 should fix this. Thanks!

Thank you for the quick fix!

**Comment by rlazo** [comment]
> > Hey Jeremy #2700 should fix this. Thanks!
> 
> Thank you for the quick fix!

np! I got all the approvals and I'm in the process of merging it.

**Comment by rlazo** [comment]
> > > Hey Jeremy #2700 should fix this. Thanks!
> > 
> > 
> > Thank you for the quick fix!
> 
> np! I got all the approvals and I'm in the process of merging it.

merged

**Pull request merged**